icubam/db/synchronizer.py

```diff
@@ -147,13 +175,23 @@ def sync_bed_counts(self, bedcounts_df, user=None):
 
 class CSVSynchcronizer(StoreSynchronizer):
   """Ingests CSV TextIO objects into datastore."""
+  def __init__(self, store_db, finess):
```

Comment:
There was the question whether we want this to generalize to other countries, in which case `finess=None` by default would be better.

Reply:
Agreed, it should be general, we should actually not use finess but icu_id or whatever the db uses as a variable name...

Reply:
agreed

Reply:
Done.